Controller/Adminhtml/Import/Ajax.php

```diff
@@ -58,30 +60,30 @@ public function execute()
 
             switch ($importType) {
                 case 'customers':
-                    $client->customerBatch($this->import->customerData->getCustomers($storeId, $chunkId, $this->import::chunkItems));
-                    $this->helper->requestLogger(__DIR__ . 'Request.log', json_encode(array('Customers' => $this->import->customerData->getCustomers($storeId, $chunkId, $this->import::chunkItems))));
-                    $this->helper->requestLogger(__DIR__ . 'Request.log', $client->customerBatch($this->import->customerData->getCustomers($storeId, $chunkId, $this->import::chunkItems)));
+                    $client->customerBatch($this->import->customerData->getCustomers($storeId, $chunkId));
+                    $this->helper->requestLogger(__DIR__ . 'Request.log', json_encode(array('Customers' => $this->import->customerData->getCustomers($storeId, $chunkId))));
+                    $this->helper->requestLogger(__DIR__ . 'Request.log', $client->customerBatch($this->import->customerData->getCustomers($storeId, $chunkId)));
                     $this->helper->requestLogger(__DIR__ . 'Request.log', '--------------------------------------');
                     $result['success'] = 'customerBatch';
                     break;
                 case 'categories':
-                    $client->categoryBatch($this->import->categoryData->getCategories($storeId, $chunkId, $this->import::chunkItems));
```

Comment:
Why are these changes in this PR?

Reply:
I found unused variable passing to methods that are not requiring it. I can revert those changes and leave them for code refactoring pr after the observers are rewritten.